_sass/_popup.scss

```diff
@@ -0,0 +1,67 @@
+.popup {
```

Comment:
Seems like you haven't activated stylelint in your editor.
Please do a `npm install` and add a corresponding stylelint plugin to your editor.

For VSCode: https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint
For Atom: https://atom.io/packages/linter-stylelint
For SublimeText: https://github.com/SublimeLinter/SublimeLinter-stylelint